### DISCUSSION TITLE
<title>Please help me on this

### DISCUSSION THREAD

**Discussion by chonma1988** [community/community#116511] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

https://github.com/CFTeamDev/CFTeamFlasher I paid 50$ for demo then isn't work they asked me to get refund 

nothing happen 

Waste money wasted the time 

please help me to get refund 

![37840](https://github.com/community/community/assets/165540057/9f0e630c-29d8-48fb-ba01-2940bfee7087)

**Comment by queenofcorgis** [comment, marked as answer]
Hi @chonma1988 , Thanks for reaching out! 

We're sorry to hear you're experiencing this issue.

GitHub is a software hosting service, we host millions of projects, but  we're not able to provide support or answers for those individual projects. GitHub Support and Community Discussions can only provide support for GitHub itself.

As no other Community Discussions team members will be able to provide additional help with these questions I am going to close this discussion.

Thank you!
